otterdog/eclipse-velocitas.jsonnet

```
@@ -198,6 +228,7 @@ orgs.newOrg('eclipse-velocitas') {
     orgs.newRepo('license-check') {
       allow_update_branch: false,
       description: "license-check",
+      has_wiki: false,
```

is there a reason to not add it everywhere?
```suggestion
      delete_branch_on_merge: true,
      description: "license-check",
      has_wiki: false,
```

---

Fixed